db/migrate/20190408215416_create_uphold_connections.rb

```diff
@@ -0,0 +1,22 @@
+class CreateUpholdConnections < ActiveRecord::Migration[5.2]
+  def change
+    create_table :uphold_connections, id: :uuid, default: -> { "uuid_generate_v4()" } do |t|
+      t.string :uphold_state_token
+      t.boolean :uphold_verified, default: false
+      t.boolean :is_member, default: false
+
+      t.uuid :uphold_id
+      t.uuid :address
+      t.belongs_to :publisher, index: true, type: :uuid
```

Comment:
Would you add a uniqueness index to this?

Reply:
https://github.com/brave-intl/publishers/pull/1825/commits/0d6e0847d3e30393b4dc8d04eeb11821ae01e544